### PR TITLE
Feat/svdb

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR adds svdb merge

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config
Comment (jemten) [inline conf/modules.config:172]:
```suggestion
        ext.args = '--notag --pass_only'
```

File: conf/modules.config
Comment (jemten) [inline conf/modules.config:181]:
```suggestion
            ext.args = '--pass_only'
            path: { "${params.outdir}/sv_caller" },
```

**Review by jemten — APPROVED** [review]
Great work Ram! Seems to me that since your last PR to modules we can move these subworkflows into the nf-core folder rather than having them in the local folder. 

Hear anything more from the nf-core team about the markdown lint fails?

**Comment by ramprasadn** [comment]
> Great work Ram! Seems to me that since your last PR to modules we can move these subworkflows into the nf-core folder rather than having them in the local folder.
> 
> Hear anything more from the nf-core team about the markdown lint fails?

True, just waiting for manta to get merged into nf-core modules. Should happen today :D 

Nope, not yet. Phil had a solution, but that triggered a few yaml lint errors so for now, I think we should just ignore it.

**Pull request merged**
